src/pages/pupils/programmes/[programmeSlug]/options.tsx

```diff
@@ -1,39 +1,19 @@
-// TESTING
-// http://localhost:3000/pupils/programmes/biology-secondary-year-11/options
-// http://localhost:3000/pupils/programmes/maths-secondary-year-10/options
-// http://localhost:3000/pupils/programmes/maths-secondary-year-11/options (should 404)
-
 import { GetStaticProps } from "next";
 
-import { PupilViewsProgrammeListing } from "@/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view";
 import {
-  ProgrammesPageProps,
   OptionsURLParams,
   getPupilOptionData,
 } from "@/pages-helpers/pupil/options-pages/options-pages-helpers";
 import { getStaticPaths as getStaticPathsTemplate } from "@/pages-helpers/get-static-paths";
+import { PupilViewsProgrammeListingProps } from "@/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view";
 
-const ProgrammesPage = ({
-  programmes,
-  baseSlug,
-  yearSlug,
-}: ProgrammesPageProps) => {
-  return (
-    <PupilViewsProgrammeListing
-      programmes={programmes}
-      baseSlug={baseSlug}
-      yearSlug={yearSlug}
-    />
-  );
-};
+export { PupilViewsProgrammeListing as default } from "@/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view";
```

Comment:
`/options.tsx` & `/options/examboard/[examboardSlug].tsx` can now just export the PupilViewsProgrammeListing as default. Far less duplication of code.